### PR TITLE
Aruha 137 metrics for http connections

### PR DESCRIPTION
**Pull request by tmuehl**
Note that this monitors only synchronous connections. For async connection this will only count for establishing the async connection via a synchronous call.

Do you have any better ideas for the names of the metrics than `openSynchronousHttpConnections` and `synchronousHttpConnections`?

@v-stepanov @CyberDem0n @rcillo @jkliff Please review

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/165" target="_blank">ReviewNinja

**Comment by rcillo** [comment]
The proposed names are clear for me. :shipit:

File: src/main/java/de/zalando/aruha/nakadi/metrics/MonitoringRequestFilter.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import java.io.IOException;
+
+public class MonitoringRequestFilter implements Filter {
+
+    private final Timer httpConnectionsTimer;
+    private Counter openHttpConnectionsCounter;
```
Comment (CyberDem0n) [inline src/main/java/de/zalando/aruha/nakadi/metrics/MonitoringRequestFilter.java]: final?

Comment (tmuehl) [inline src/main/java/de/zalando/aruha/nakadi/metrics/MonitoringRequestFilter.java]: Yes!

**Pull request merged**